### PR TITLE
fix filter_alleles doc in dataset.py

### PR DESCRIPTION
**Pull request by jbloom22**

### CODE REVIEW

File: python/hail/dataset.py [hunk 1]
```diff
@@ -1408,37 +1408,33 @@ def write(self, output, overwrite=False):
...
-        evaluated for each alternate allele.  It is not evaluated for
-        the reference (i.e. ``aIndex`` will never be zero).
+        evaluated for each alternate allele, but not for
+        the reference allele (i.e. ``aIndex`` will never be zero).
 
         **Example**
 
-        Remove alternate alleles whose allele count is zero and
-        updates the alternate allele count annotation with the new
+        To remove alternate alleles with zero allele count and
```
Comment (jigold) [inline python/hail/dataset.py]: Replace "To remove" with "Remove"

File: python/hail/dataset.py [hunk 2]
```diff
@@ -1534,12 +1527,14 @@ def filter_alleles(self, condition, annotation=None, subset=True, keep=True, fil
...
+            
+        :param bool subset: If true, subsets PL and AD, otherwise downcodes the PL and AD.
             Genotype and GQ are set based on the resulting PLs.
-        :param bool keep: Keep variants matching condition
-        :param bool filter_altered_genotypes: If set, any genotype call that would change due to filtering an allele
-            would be set to missing instead.
+            
+        :param bool keep: If true, keep variants matching condition
+        
+        :param bool filter_altered_genotypes: If true, genotypes that contain filtered-out alleles are set to missing.
```
Comment (jigold) [inline python/hail/dataset.py]: Could you please add the :return and :rtype while you have this pull request open?

File: python/hail/dataset.py (hunk 1 above)
Comment (jbloom22) [inline python/hail/dataset.py]: I've been using "To" everywhere. Let's find concensus as group.

File: python/hail/dataset.py (hunk 2 above)
Comment (jbloom22) [inline python/hail/dataset.py]: Done, back to you!

**Pull request merged**